### DISCUSSION TITLE
`dep ssh` stuck in terminal

### DISCUSSION THREAD

**Discussion by isaacdarcilla** [deployphp/deployer#3731] [Help needed]
When running `dep ssh -vvv`, it does not log any message in terminal and get stuck.

![image](https://github.com/deployphp/deployer/assets/22732118/b69306c4-3c6f-4632-87c2-dfc640e13f75)

But when running `ssh` directly, it works.

![image](https://github.com/deployphp/deployer/assets/22732118/4c8f2136-f11c-4d09-8a6b-2acd08517fb7)

Any idea why `dep ssh` is stuck?

**Comment by jasperf** [comment]
You do not show any details using `dep ssh -vvv`. Would be better if you paste full log as text minus the sensitive data like username, server name and ip addresses.

**Reply by isaacdarcilla** [reply]
There's actually no logs. This can be possibly related to network issue.
